### PR TITLE
feat: add single active consumer argument support

### PR DESCRIPTION
**Pull request by w1zart**
Adds SAC argument support for queue

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
-import { AMQPOptions } from './adapters/amqp-node';
+import { AMQPOptions, AMQPQueueOptions } from './adapters/amqp-node';
 import { Logger } from './logger';
 import { MessageHandler } from './message';
 
 export interface CreateServiceOptions {
   serviceName: string;
   logger: Logger;
   connectOptions: AMQPOptions;
+  queueOptions?: AMQPQueueOptions;
```
Comment (sgrigorev) [inline src/index.ts:10]: Could you please update the readme file with new option?

Comment (w1zart) [reply src/index.ts:10]: Done

**Comment by w1zart** [comment]
@semurgx @leoShabalkin

File: README.md [hunk 2]
```diff
@@ -31,10 +31,17 @@ const client = createClient({
     amqps: true,
     frameMax: 8192,
   },
+  queueOptions: {
+    args: {
```
Comment (alexkvak) [inline README.md:35]: why such additional key `args` is needed?

Comment (w1zart) [reply README.md:35]: In order to have a dedicated args object, just for future support of other options, not to mix them up with the args group

Comment (alexkvak) [reply README.md:35]: but you can support new options in `queueOptions` directly. You do not need any additional wrapper such as `args`

**Review by alexkvak — APPROVED** [review]

**Review by leoShabalkin — APPROVED** [review]

**Review by sgrigorev — APPROVED** [review]

**Pull request merged**